deeplearning4j-core/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java

```diff
@@ -847,7 +847,7 @@ public INDArray params(boolean backwardOnly) {
         List<INDArray> params = new ArrayList<>();
         for (Layer layer: getLayers()){
             INDArray layerParams;
-            if( layer instanceof BasePretrainNetwork && backwardOnly)
+            if(layer instanceof BasePretrainNetwork)
                 layerParams = ((BasePretrainNetwork) layer).paramsBackprop();
```

Comment:
@AlexDBlack relooking at this realized why we have it because BasePretrainingNewtork's method is labeled paramsBackprop(). I'm going to change it to params() to work with this PR but give a shout out if there is a reason to reverse back.